common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1356,6 +1356,17 @@ private boolean isTooLate(
         !appShuffleMergePartitionsInfo.shuffleMergePartitions.containsKey(reduceId);
     }
 
+    /**
+     * If necessary, look for the ignored bock bytes and capture them in pushMergeMetrics.
+     */
+    private void lookForIgnoredBlockBytesIfNecessary() {
```

Comment:
nit: rename to `updatedIgnoreBytesWithDeferredBufs` or `deferredBufsIgnored` or something similar.

Reply:
Thanks for the suggestions, Chandni. Renamed it to `updateIgnoredBytesWithDeferredBufs` based on the former, as it sounds more smooth to me. Hopefully it looks better. Please let me know otherwise.